piplicenses.py

```diff
@@ -166,7 +166,8 @@ def get_pkg_included_file(pkg, file_names):
                                                        f))))
          for f in file_names]
         for test_file in patterns:
-            if os.path.exists(test_file):
+            if os.path.exists(test_file) and \            
```

Comment:
The end of the line continuation cannot contain spaces.

Please remove the spaces behind the `\`

Reply:
Sorry @raimon49 don't intend to offend you but I didn't planned to spend hours on this SLOC. If you might be too busy to copy-past this 2 lines in to code in the style you prefer. If not it's giving me a hard time to understand why you keep pinging me on this.
I did not intend to apply to anything I wanted to help out as the conversation in #121 apparently has stalled leaving a breaking bug for windows users.

I removed the surplus spaces. This will be the final edit from my side.

In the future I definitely won't step in. I do not fancy a membership in your exclusive club of contributors

Reply:
@eUgEntOptIc44 It appears that my request for changes has offended you. I am sorry.

We apply automatic checks to ensure that we adhere to the code style set forth by the Python community.
https://github.com/raimon49/pip-licenses/blob/master/CONTRIBUTING.md

And I am willing to accept as much code as possible from those who open P-R. Since this process seemed cumbersome to you, I have taken it up myself and released it. #123 

Thank you for your contribution and challenge.